docs/book/component-guide/data-validators/evidently.md

```diff
@@ -414,7 +420,7 @@ The Evidently Data Validator implements the same interface as do all Data Valida
 All you have to do is call the Evidently Data Validator methods when you need to interact with Evidently to generate data reports or to run test suites, e.g.:
 
 ```python
-from typing_extensions import Annotated  # or `from typing import Annotated on Python 3.9+
+from typing_extensions import Annotated  # or `from typing_extensions import Annotated on Python 3.9+
```

Comment:
The comment is the same as the actual import 😄 and I think we should use whatever is recommended for Python 3.9+.

Reply:
There are a few instances where the same import shows up with the same comment. I would fix those as well.

Reply:
Also if possible, I would use `from typing import Annotated` as we are 3.9+ anyhow.